plugin/storage/integration/cassandra_test.go

```diff
@@ -64,13 +64,20 @@ func (s *CassandraStorageIntegration) initializeCassandraFactory(t *testing.T, f
 func (s *CassandraStorageIntegration) initializeCassandra(t *testing.T) {
 	f := s.initializeCassandraFactory(t, []string{
 		"--cassandra.keyspace=jaeger_v1_dc1",
+		"--cassandra-archive.keyspace=jaeger_v1_dc1",
```

Comment:
you should use a different name vs. the primary keyspace

Reply:
In the config.yaml file, Cassandra main and archive have the same Keyspace configuration (jaeger_v1_dc1), using any other configuration is throwing out an error, refusing to connect?

Reply:
In the script that prepares Cassandra we may need to run schema creation step twice, with different keyspaces.

Reply:
Can you PTAL? Is this feasible?